R/Tidy_names_types_factors.R

```diff
@@ -35,10 +38,14 @@
 #' @export
 #'
 #' @author J. Peter Marquardt
-apply_data_dictionary <- function(data, data_dictionary) {
+apply_data_dictionary <- function(data, data_dictionary, print_coerced_NA = TRUE) {
 
   assertive.types::is_data.frame(data)
   assertive.types::is_data.frame(data_dictionary)
+  assertive.types::is_a_bool(print_coerced_NA)
```

Comment:
Need to update assertive.types dependencies

Reply:
fixed